test/viz/taxa_dynamics.jl

```diff
@@ -0,0 +1,36 @@
+using WGLMakie, GeoMakie, GraphMakie
+
+using ADRIA
+
+if !@isdefined(TEST_RS)
+    const TEST_DOM, TEST_N_SAMPLES, TEST_SCENS, TEST_RS = test_rs()
+end
+
+@testset "taxonomy" begin
```

Comment:
You've removed `taxa` elsewhere, maybe a chance to rename this `functional_groups`?